CUDLR/Scripts/Console.cs

```diff
@@ -14,7 +15,25 @@ struct QueuedCommand {
   }
 
   public class Console {
-
+     
+     /* Struct containing console message and color data */
+	 struct ConsoleMessage
+      {
+          public string message;
+          public Color32 color;
```

Comment:
Actually reviewing this more, I think that it's better to use Color32 consistently between Server.cs and Console.cs. For the purposes of this asset, color fidelity is not needed, and Color32 seems to be the better choice due to the following:
- Color32 stores its color data using 4 bytes, whereas Color uses 4 floats
- Color has some nice static defaults that can be used, however you can still use these for Color32. For example:
Color32 myColor = Color.magenta; 
This is valid and is implicitly cast to a Color32 without issue. 

Also, user defined presets from within Unity can be applied to either Color or Color32 objects. Given this, I went ahead and updated the MessageColors struct in Server.cs to use Color32 as well. 

Please let me know if you think that there may be other benefits to using Color over Color32 in this case.